components/list/list.css

```diff
@@ -0,0 +1,53 @@
+.list-no-bullet {
+  list-style-type: none;
+}
+
+.list-no-bullet > li {
+  padding: 4px 0px;
+}
+
+.checked-list {
+  list-style: none;
+}
+
+.checked-list-item {
+  margin: 8px 0px;
```

Comment:
try using rem instead of px.

Reply:
hey, I am using rem for anything above 10 px.